deliverables/README.md

```diff
@@ -21,48 +21,117 @@ For projects on Code Academy, or other external exercises, please just paste a l
+The default styles are inherited, ie passed down to a younger generation, from the parent element to the child element. 
+
 What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use text-align or through commands like center me.
 
+What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+You can use in line, external or embedded css. 
 ---
 ### Section 3: Git and Github  
-What is Git?  
+What is Git?
+Git is a system that allows revision control for software development. 
+
 What is SCM?  
+Source code manager. Used to manage source code and track revisions in software.  
+
 What is a VCS?  
-Why is Git useful for a developer?  
+Veritas Cluster Server provides cluster capabilities to systems running other applications. 
+
+Why is Git useful for a developer?
+Git stores and gives feedback to all of your code and allows you to maintain an organized workspace. 
+
 Why is Git useful for a team of developers?  
-How do you create a new Git repository for a project locally?  
+Git allows a team to work together using all of the same files. Editing is done live so everyone can see the changes and add to them. 
+
+How do you create a new Git repository for a project locally? 
+git init command creates a new project. 
+
 How do you create it on Github?  
+You can either use gits built in command line or a regular command line. 
+
 How do you commit changes?  
+You commit chanes at the bottom of the editing page. You can either commit to master or commit to other places. 
+
 What is the difference between staging and committing changes?  
+The staging is like a loading dock where you can chose what changes you want to commit. Commmitting is saying, I want these changes actually applied. 
+
 What is the difference between committing your changes and pushing them to Github?  
+Commiting records changes to the local repo while pushing updates remote repositories. 
+
 What is the command to check the status of your current repo in git?  
-What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
+git-status
+
+What is the command to see the history of your previous commits (from the command line)?  
+git-log 
+
 How can you look through your historical commits on the Github website?  
+Click on the number of commits button at the top of your profile in github. Ex. 34 commits. Click that and it will show you your history.
+
 What is a "Merge"?  
+A merge is when you join two or more development histories together. 
+
 What is a "Pull Request"?  
+Pull requests allow you to tell others about the changes you have pushed to a repo.
+
 What is "Forking" a repo?  
+A fork is a copy of a repo where you can make your own changes without affecting the original. 
```

Comment:
... on Github.  "Fork" is a Github-specific concept.  Basically it's "clone" but specifically cloning the project from their account to your account, under the same project name.